data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
```

Comment:
separate label or just need our uniform/preferred title?

Also, will we need an array for 'alternate names' that would be separate from a computed list of attested names in the mss?